duckduckgo/Android
# Suppress system AutofillService save prompts to prevent it overlapping in-browser save prompts

Task/Issue URL: https://app.asana.com/0/608920331025315/1205657473475434/f 

### Description
Prevent an overlap occurring between the system AutofillService save prompt and the in-browser save prompt. Note, this isn't fool proof and there is at least one known site where it doesn't prevent both showing, but it's a UX improvement on majority of sites.

### Steps to test this PR

To test this properly, ensure your Android version is >=  Android 8.0 (API 26) and that you have a system-level Autofill provider set up (otherwise you would never have seen the system prompt anyway, so the suppression isn't being tested). You can verify this here:
- Visit settings, languages & input, autofill service (or you might have to search for it in settings if OEM has put it somewhere else)

<img src="https://github.com/duckduckgo/Android/assets/1336281/75ca068b-549d-4cca-a79a-f029af76b86e" width="30%" />

<hr />

**Steps**
- [x] Install `internal` app build
- [x] Visit https://fill.dev/form/login-simple; enter username, password and tap login button
- [x] Verify you only see our save prompt, and not the system (e.g., Google Password Manager) save prompt

### UI changes

![combined](https://github.com/duckduckgo/Android/assets/1336281/645e6943-7c64-47fd-88b1-f6dfedd64e5c)
